sdk/java/pulumi/src/test/java/com/pulumi/provider/internal/infer/ComponentAnalyzerTests.java

```diff
@@ -32,7 +32,8 @@ void testGenerateSchemaWithoutMetadata() {
         var expected = new PackageSpec()
             .setName("infer")
             .setDisplayName("infer")
-            .setVersion(null);
+            .setVersion(null)
+            .setNamespace("pulumi");
```

Comment:
Before this PR there would not have been any namespace set, so this will test it.